offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/OfferToPDFConverter.groovy

```diff
@@ -234,12 +252,34 @@ class OfferToPDFConverter implements OfferExporter {
             String elementId = "product-items" + "-" + tableCount
             htmlContent.getElementById("item-table-grid").append(ItemPrintout.tableHeader(elementId))
             htmlContent.getElementById("item-table-grid")
-                    .append(ItemPrintout.tableFooter(offer.overheadRatio))
+                    .append(ItemPrintout.tableFooter(offer.overheadRatio, offer.getSelectedCustomerAffiliation()))
         } else {
             //otherwise add total pricing to table
             htmlContent.getElementById("item-table-grid")
-                    .append(ItemPrintout.tableFooter(offer.overheadRatio))
+                    .append(ItemPrintout.tableFooter(offer.overheadRatio, offer.getSelectedCustomerAffiliation()))
+        }
+    }
+
+    void setTaxationStatement() {
+        if(!offer.getSelectedCustomerAffiliation().country.equals(countryWithVAT)) {
+            htmlContent.getElementById("vat-cost-applicable").text("Taxation is not applied to offers outside of ${countryWithVAT}")
+        }
+
+    }
+
+    void setTaxationRatioInSummary() {
+        DecimalFormat decimalFormat = new DecimalFormat("#%")
+        double taxRatio = determineTaxCost(offer.getSelectedCustomerAffiliation().getCountry(), offer.getSelectedCustomerAffiliation().getCategory())
+        String taxPercentage = decimalFormat.format(taxRatio)
+        htmlContent.getElementById("total-taxes-ratio").text("VAT (${taxPercentage})")
+    }
+
+    // Apply VAT only if the offer originated from Germany and it's affilation category is non-internal
+    static double determineTaxCost(String country, AffiliationCategory category) {
+        if(country.equals(countryWithVAT) && !category.equals(noVatCategory)) {
+            return VAT
         }
+            return 0
     }
```

Comment:
@KochTobi i adapted your suggestion a little bit and introduced the determineTaxCost method which already returns the tax cost value.